### ISSUE TITLE
rnet时生成的rnet_epoch_1.pt-rnet_epoch_10.pt人脸一个也检测不到

### ISSUE THREAD

**Issue by sc199505** [Sierkinhane/mtcnn-pytorch#3]
你好，问下按照你的步骤在wider 数据集上进行训练，在选用pnet_epoch_10.pt 去训练rnet时生成的rnet_epoch_1.pt-rnet_epoch_10.pt人脸一个也检测不到。

**Comment by Sierkinhane** [comment]
不明白你讲的是什么，先生成pnet的训练数据，训练pnet，用训练好的pnet模型去生成rnet的训练数据...

**Comment by sc199505** [comment]
你好问题是这样的，我用自己训练好的pnet_epoch_6.pt和rnet_epoch_10.pt,和你之前训练好的onet_epoch.pt进行检测的时候，一张人脸也检测不到，但当我把rnet_epoch_10.pt更换成你之前训练好的rnet_epoch.pt时就可以检测到人脸。

**Comment by zhangshitoday** [comment]
I meet the same question, the r net seems non-convergence

**Comment by TTTREE** [comment]
同问，请问您解决这个问题了吗？

**Comment by zhangshitoday** [comment]
As for me, the saved model except the last one are all worked, namely rnet_epoch_9.pt is all right.

**Comment by TTTREE** [comment]
> As for me, the saved model except the last one are all worked, namely rnet_epoch_9.pt is all right.

请问您的anno_train_test.txt是怎么来的，我怀疑是数据处理阶段的问题，因为其他代码我都没有动过，谢谢

**Comment by zhangshitoday** [comment]
Does anno_train_test.txt mean the anno file of origin image set? it is image path and ground truth box in every line.

**Comment by TTTREE** [comment]
> 
> 
> Does anno_train_test.txt mean the anno file of origin image set? it is image path and ground truth box in every line.

Yes, but when I uesd the whole train data , which is the same as anno_train.txt, the result of RNET can not detect correctly. And then , I chose 5800 pics from anno_train.txt for training Rnet and  Onet . It works . So I guess maybe there should be some data-preprocess methods for the whole train data, to remove some blurred, special pictures. 
Btw, I tested the result model on WIDER-FACE, and a seemingly not very good result was achieved, even when I uesd the pre-trained model which is provided by the author.  So I think maybe something wrong.

**Comment by westnight** [comment]
> , but when I uesd the whole train data , which is the same as anno_train.txt, the result of RNET can not

我碰到另一个问题，整个模型对小人脸的检测效果还可以，但是对大人脸非常差，你碰到了吗

**Comment by hamhanry** [comment]
for the Rnet to make it conver, try to change the parameter such as lr and batch size.
i changed mine into lr = 0.001 and batch size of 128

**Comment by hamhanry** [comment]
> I meet the same question, the r net seems non-convergence

for the Rnet to make it conver, try to change the parameter such as lr and batch size.
i changed mine into lr = 0.001 and batch size of 128
